module/build.sbt

```diff
@@ -56,11 +51,11 @@ ReleaseKeys.releaseProcess := Seq[ReleaseStep](
   commitReleaseVersion,
   tagRelease,
   ReleaseStep(
-    action = state => Project.extract(state).runTask(PgpKeys.publishSigned, state)._1,
+    action = Command.process("publishSigned", _),
     enableCrossBuild = true
   ),
   setNextVersion,
   commitNextVersion,
-  ReleaseStep(state => Project.extract(state).runTask(SonatypeKeys.sonatypeReleaseAll, state)._1),
+  ReleaseStep(action = Command.process("sonatypeReleaseAll", _)),
```

Comment:
Thanks - I think your conf is a bit neater, so I've changed to that in 2ef3d5d20